### DISCUSSION TITLE
Show status of entity in the tab

### DISCUSSION THREAD

**Discussion by mogim0gi** [kinghat/tabbed-card#141] [Feature Requests]
Hi, using this card and loving it, however i have a suggestion, can you make a customisable option to display status of card with color? in my example i would like to show which thermostats are currently heating: 

<img width="390" alt="image" src="https://github.com/kinghat/tabbed-card/assets/64898710/d25a55c0-f713-47a9-be5f-048ca3d7b5ae">

**Comment by ildar170975** [comment]
May be easily achieved by card-mod.
Ask in the Community card-mod thread.

**Reply by kinghat** [reply]
not sure which is the appropriate thread but here are two:
https://community.home-assistant.io/t/card-mod-add-css-styles-to-any-lovelace-card/120744
https://community.home-assistant.io/t/card-mod-super-charge-your-themes/212176

**Reply by mogim0gi** [reply]
Okay, i was hoping this is a feature you would be willing to implement, as this gives so much more use possiblities to the card

**Reply by ildar170975** [reply]
Follow the 1st link, 1st post of the thread -> link at the bottom -> styles for tabbed-card.
